<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config::AddDSMComboDialog</class>
 <widget class="QDialog" name="config::AddDSMComboDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add Sensor</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1</y>
     <width>202</width>
     <height>154</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer_top">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>168</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="DSMNameLayout">
      <item>
       <widget class="QLabel" name="DSMNameLabel">
        <property name="text">
         <string>DSM Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DSMNameText"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="DSMIdLayout">
      <item>
       <widget class="QLabel" name="DSMIdLabel">
        <property name="text">
         <string>DSM Id</string>
        </property>
        <property name="buddy">
         <cstring>DSMIdText</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DSMIdText">
        <property name="toolTip">
         <string extracomment="Numeric Id - default is know unique - must be unique"/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="LocationLayout">
      <item>
       <widget class="QLabel" name="LocationLabel">
        <property name="text">
         <string>Location</string>
        </property>
        <property name="buddy">
         <cstring>LocationText</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LocationText">
        <property name="toolTip">
         <string extracomment="Where is DSM located in Plane"/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_bottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>168</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>DSMIdText</tabstop>
  <tabstop>LocationText</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>config::AddDSMComboDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>config::AddDSMComboDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newSensor(QString)</slot>
  <slot>setDevice(int)</slot>
 </slots>
</ui>
